shared-bindings/os/__init__.c

```diff
@@ -166,6 +166,20 @@ MP_DEFINE_CONST_FUN_OBJ_1(os_rmdir_obj, os_rmdir);
 //| def stat(path: str) -> Tuple[int, int, int, int, int, int, int, int, int, int]:
 //|     """Get the status of a file or directory.
 //|
+//|        Returns a tuple with the status of a file or directory in the following order:
+//|
+//|
+//|        * ``st_mode`` -- File type, regular or directory
+//|        * ``st_ino``  -- Unpopulated
+//|        * ``st_dev`` -- Unpopulated
+//|        * ``st_nlink`` -- Unpopulated
+//|        * ``st_uid`` -- Unpopulated
+//|        * ``st_gid`` -- Unpopulated
```

Comment:
When you say `Unpopulated`, do you mean it is zero? Could you say that explicitly?